project_templates/stack_package/example_pythononly/ups/example_pythononly.cfg

```diff
@@ -0,0 +1,24 @@
+# -*- python -*-
+
+import lsst.sconsUtils
+
+# Dependencies that provide header files and or libraries should be included here.
+# Pure-Python dependencies do not need to be included.
+# Packages that use pybind11 or boost_tests should declare them as build dependencies.
+# Otherwise, the rules for which packages to list here are the same as those for
+# table files.
+dependencies = {
+    "required": ["utils", "ndarray"],
+    "buildRequired": ["boost_test", "pybind11", "ndarray", "pex_exceptions"],
+    "optional": [],
+    "buildOptional": [],
+}
+
+# For packages that build a C++ library and a Python module, the below should be sufficient.
+# Pure-Python packages should set headers=[], libs=[] (not libs=None). and hasSwigFiles=False.
+# For more information, see the sconsUtils Doxygen documentation.
+config = lsst.sconsUtils.Configuration(
+    __file__,
+    headers=["lsst/TMPL.h"],
```

Comment:
I thought I had added a comment about this... I think this is in error.  The stand-in header file has been copied to the correct place already.  I missed this in the first example because it things that file was renamed.